### PR TITLE
Fix the increment workflow

### PR DESCRIPTION
**Pull request by danielle-casella-adams**
Fix the increment workflow so it can handle the scenario where there are no existing tags.  Since the tag and the push are using github-script now, I've just combined them into the same step.

I thought the previous bash syntax (`tag=$(git describe --tags --abbrev=0) || tag=1.0.-1`) with the || would help but Actions still considered it a fatal error and it did not work correctly.  

There are a few npm updates because the pre-commit hook wasn't running correctly.

### CODE REVIEW

File: .github/workflows/increment-version-on-merge.yml [hunk 1]
```diff
@@ -16,30 +16,34 @@ jobs:
...
+              tag = tag.replace('\n', '').trim();
+              core.info(`The latest tag is: ${tag}`);
+              const majorMinorPatch = tag.split('.');
+              const patch = parseInt(majorMinorPatch[2]) + 1;
+              newTag = `${majorMinorPatch[0]}.${majorMinorPatch[1]}.${patch}`
+              core.info(`The new tag is: ${newTag}`);
+            }
+            catch (error) {
+              newTag = 'v1.0.0';
+              core.info('An error occurred getting the tags for the repo.  It most likely does not have any tags so use 1.0.0.');
```
Comment (JosephDSchwartz) [inline .github/workflows/increment-version-on-merge.yml:39]: Should the end read something like "It most likely does not have any tags to use. Defaulting to 1.0.0."? It _kinda_ makes sense right now, but is a little confusing.

File: package.json [hunk 2]
```diff
@@ -22,8 +22,9 @@
     "@actions/core": "^1.2.7"
   },
   "devDependencies": {
-    "esbuild": "^0.12.4",
+    "esbuild": "^0.12.8",
     "husky": "^4.3.8",
+    "install": "^0.13.0",
```
Comment (JosephDSchwartz) [inline package.json:27]: Does this get used somewhere behind the scenes? I don't think I've ever used this package so I'm not familiar with it.

Comment (danielle-casella-adams) [reply package.json:27]: When I removed esbuild and then put esbuild back in it added that one for me.

File: .github/workflows/increment-version-on-merge.yml (hunk 1 above)
Comment (danielle-casella-adams) [reply .github/workflows/increment-version-on-merge.yml:39]: Yes I can update the wording.

File: package.json (hunk 2 above)
Comment (JosephDSchwartz) [reply package.json:27]: Huh, weird! It's not hurting anything so... 🤷‍♂️

**Review by JosephDSchwartz — APPROVED** [review]

**Pull request merged**